### PR TITLE
added 'Group14' class

### PR DESCRIPTION
**Pull request by ecaliskanunal**

### CODE REVIEW

**Review by fikretellek — APPROVED** [review]

File: src/test/java/Group14.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class Group14 {
+    public static void main(String[] args) {
+        System.out.println("Hello Group 14!");
```
Comment (ademcigdem) [inline src/test/java/Group14.java:3]: Group-14 is good

**Review by ademcigdem — APPROVED** [review]

**Review by busragokner — APPROVED** [review]
Good job

File: src/test/java/Group14.java (hunk 1 above)
Comment (ecaliskanunal) [reply src/test/java/Group14.java:3]: Thanks for the comment!

**Comment by CengizKoch** [comment]
well done

**Comment by GokceKochan** [comment]
well done

**Comment by GokceKochan** [comment]
well done

**Pull request merged**